ts-sdk/src/utils/index.ts

```diff
@@ -9,3 +9,11 @@ export const generateSalt = () => {
   }
   return toHex(rawSalt) as Hex
 }
+
+export const generateSaltAptos = () => {
+  const rawSalt = new Uint8Array(14)
+  for (let i = 0; i < rawSalt.length; i++) {
+    rawSalt[i] = Math.floor(Math.random() * 256)
+  }
+  return toHex(rawSalt) as Hex
+}
```

Comment:
make rpcType (`evm | cosmos | aptos`) a parameter to this function, if aptos take 14, else take 32. unify the implementations